src/main.js

```diff
@@ -38,7 +150,12 @@ export async function applyAndMerge(planDefinition, patientReference=null, resol
  * @param {Object} aux - Auxiliary resources and services
  * @returns {Promise<Object[]>} Array of resources: CarePlan, RequestGroup, and otherResources
  */
-export async function applyPlan(planDefinition, patientReference=null, resolver=null, aux={}) {
+export async function applyPlan(
```

Comment:
I tested this PR by upgrading the encender dependency in ccsm-cds-dashboard to reflect this branch.

I'm seeing the following error when I test:
```
main.js:13 Uncaught (in promise) TypeError: (0 , module__WEBPACK_IMPORTED_MODULE_1__.createRequire) is not a function
   at main.js:13:1
   at executeCQL (main.js:16:1)
   at applyPlan (main.js:156:1)
   at async applyCds (index.js:78:1)
```
This error is appearing when I test with both the local FHIR patients in the `tests-fhir` path, as well as when launching via SMART on FHIR using https://launch.smarthealthit.org/.

Reply:
This is the code that Mike and I have both been testing against.  But we have also been doing while having the dashboard pull in a branch of cql-execution.  That may very well be the issue, it's a reason I really don't really like using git urls for package dependencies

Reply:
CQL execution still gets brought in by CQL Worker, right? I tested by:
- changing the `encender` dependency in package.json to `"github:ccsm-cds-tools/encender#add_cql_worker"`
- clearing my `node_modules` folder
- running `npm install`
- running `npm upgrade encender` to update to the latest version of this encender branch

When doing that, I was left with `cql-execution` version `3.0.1` in my `node_modules` folder, which according to `package-lock.json` was brought in through `cql-worker`.

Reply:
I just looked into this a bit.  I think it depends on what dashboard branch you may have tested against.  That is an error when the worker factory is not passed to encender.  That code may be commented out in your version of the dashboard so it needs to be uncommented and the worker factory passed into the aux variable

Reply:
Good catch. I was using the `pilot` branch of [ccsm-cds-dashboard](https://github.com/ccsm-cds-tools/ccsm-cds-dashboard/tree/pilot). But in https://github.com/ccsm-cds-tools/ccsm-cds-dashboard/pull/58, we removed the `WorkerFactory`, since it wasn't used in the `cql-execution` branch of Encender that we were using.

I just created a new branch, [restore-worker-factory](https://github.com/ccsm-cds-tools/ccsm-cds-dashboard/tree/restore-worker-factory) that restores the commented out WorkerFactory code and updates encender to use this branch. Testing with that dashboard branch was successful for both the local FHIR patients and the SMART on FHIR patients! So once we merge this in and update the encender dependency in the dashboard, we'll just have to remember to add the WorkerFactory code back to the Dashboard.